gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -505,6 +505,7 @@
     [150000, 'array_length', 'INT', ['ANY_ARRAY'], 'ArrayFunctions::array_length'],
     [150001, 'array_append', 'ANY_ARRAY', ['ANY_ARRAY', 'ANY_ELEMENT'], 'ArrayFunctions::array_append'],
     [150002, 'array_contains', 'BOOLEAN', ['ANY_ARRAY', 'ANY_ELEMENT'], 'ArrayFunctions::array_contains'],
+    [150002, 'array_remove', 'BOOLEAN', ['ANY_ARRAY', 'ANY_ELEMENT'], 'ArrayFunctions::array_remove'],
```

Comment:
ok, i modify it to different functions

Reply:
Since `array_remove` will return a new array, the return type should be `ANY_ARRAY` instead of `BOOLEAN`, i.e,
`[150003, 'array_remove', 'ANY_ARRAY', ['ANY_ARRAY', 'ANY_ELEMENT'], 'ArrayFunctions::array_remove']`